sdk/head_tracker.cc

```diff
@@ -34,49 +34,52 @@ void GetViewporOrientationRotation(
   static const Rotation kEkftoHeadTrackerLandscapeLeft =
           Rotation::FromQuaternion(Rotation::QuaternionType(0.5, -0.5, -0.5, 0.5));
   static const Rotation kSensorToDisplayLandscapeLeft =
-          Rotation::FromAxisAndAngle(Vector3(0, 0, 1), M_PI / 2.);
+          Rotation::FromAxisAndAngle(Vector3(0., 0., 1.), M_PI / 2.);
 
   // This is the same than initializing the rotation from
   // Rotation::FromYawPitchRoll(M_PI / 2., 0, M_PI / 2.).
   static const Rotation kEkftoHeadTrackerLandscapeRight =
           Rotation::FromQuaternion(Rotation::QuaternionType(0.5, 0.5, 0.5, 0.5));
   static const Rotation kSensorToDisplayLandscapeRight =
-          Rotation::FromAxisAndAngle(Vector3(0, 0, 1), -M_PI / 2.);
+          Rotation::FromAxisAndAngle(Vector3(0., 0., 1.), -M_PI / 2.);
 
   // This is the same than initializing the rotation from
   // Rotation::FromYawPitchRoll(M_PI / 2., M_PI / 2., M_PI / 2.).
   static const Rotation kEkftoHeadTrackerPortrait =
           Rotation::FromQuaternion(Rotation::QuaternionType(0.707107, 0, 0, 0.707107));
```

Comment:
Please use float literals (see https://en.cppreference.com/w/cpp/language/floating_literal). Plese fix here and there.